### PR TITLE
Add an endpoint and fix some bugs

### PR DESCRIPTION
**Pull request by sim0629**
For #3, downloading on android was tested and it works well.

### CODE REVIEW

**Comment by simnalamburt** [comment]
@sim0629 @devleoper I'm going to relicense this repository under dual MIT/Apache-2.0. To agree to relicensing, comment with :
```
I license past and future contributions under the dual MIT/Apache-2.0 license,
allowing licensees to chose either at their option.
```

## Why?
The MIT license requires reproducing countless copies of the same copyright header with different names in the copyright field, for every MIT library in use. The Apache license does not have this drawback. However, this is not the primary motivation for me to do this. The Apache license also has protections from patent trolls and an explicit contribution licensing clause. However, the Apache license is incompatible with GPLv2. This is why Rust is dual-licensed as MIT/Apache (the "primary" license being Apache, MIT only for GPLv2 compat), and doing so would be wise for this project.

Some ask, "Does this really apply to binary redistributions? Does MIT really require reproducing the whole thing?" I'm not a lawyer, and I can't give legal advice, but some Google Android apps include open source attributions using this interpretation. Others also agree with it.

But, again, the copyright notice redistribution is not the primary motivation for the dual-licensing. It's stronger protections to licensees and better interoperation with the wider ecosystem.

File: lib/kollus.rb [hunk 1]
```diff
@@ -51,13 +60,14 @@ def token; @token end
...
 
-  def upload(title = nil, expire_time = 600, encrypted = true, audio = false)
+  def upload(category_key = nil, title = nil, expire_time = 600, encrypted = true, audio = false)
+    tries ||= 3
     api_uri = URI('http://api.kr.kollus.com/0/media_auth/upload/create_url.json?access_token=' + @token)
     params = {
       # 값의 범위는 0 < expire_time <= 21600 입니다. 빈값을 보내거나 항목 자체를 제거하면 기본 600초로 설정됩니다.
       expire_time: expire_time,
       # 업로드한 파일이 속할 카테고리의 키(API를 이용하여 확득 가능)입니다. 빈값을 보내 거나 항목 자체를 제거하면 '없음'에 속합니다.
-      category_key: 'bw2z2ngbkatkqx77',
```
Comment (simnalamburt) [inline lib/kollus.rb:60]: @sim0629 Is this category key OK to be in the source code history? If I have to delete it from the history, I will remove it with force-push after merging this PR.

**Review by yosangwon — APPROVED** [review]
I license past and future contributions under the dual MIT/Apache-2.0 license,
allowing licensees to chose either at their option.

**Comment by sim0629** [comment]
I license past and future contributions under the dual MIT/Apache-2.0 license,
allowing licensees to chose either at their option.

**Review by simnalamburt — APPROVED** [review]

File: lib/kollus.rb (hunk 1 above)
Comment (sim0629) [reply lib/kollus.rb:60]: It's okay. I think it is useless unless one knows the api key corresponding to the category.

Comment (simnalamburt) [reply lib/kollus.rb:60]: Thank you for your explanation

**Pull request merged**